dogfood/contents/nix.hash

```diff
@@ -0,0 +1,2 @@
+7f76642cfe331925935c526e068788c30213e36df85b20214d517d1a2fdf3954  flake.nix
+2358e758b2369c744a3f38d5b6cd3a71e79fcb99bdbb3bdeb8d3a520cf46f19f  flake.lock
```

Comment:
Can this live in the repo root if it is consumed by someone running `nix develop` in the repo root?
Or is it only useful for building the Docker image?